sdk/typescript/src/client/client.ts

```diff
@@ -738,13 +739,23 @@ export class SuiClient {
 		});
 	}
 
-	async resolveNameServiceNames(
-		input: ResolveNameServiceNamesParams,
-	): Promise<ResolvedNameServiceNames> {
-		return await this.transport.request({
-			method: 'suix_resolveNameServiceNames',
-			params: [input.address, input.cursor, input.limit],
-		});
+	async resolveNameServiceNames({
+		format = 'dot',
+		...input
+	}: ResolveNameServiceNamesParams & {
+		format?: 'at' | 'dot';
```

Comment:
I don't love this naming, It's based on whats used in the rust code, but would love a better suggestion

Reply:
What would be your suggestion? We already named it like this in GraphQL too though!

Reply:
I didn't have anything better, so if it's consistent with GraphQL probably better to just leave it!